packages/react/src/useField.js

```diff
@@ -256,6 +263,16 @@ const useField = ({
           value
           === ''
         )
+        && (
+          (
+            inputValue
+            == null
+          )
```

Comment:
Are we using `== null` instead of `=== null` to accommodate `undefined`?

If so, I prefer being explicit and adding `=== undefined` as well. I know it's an extra check, but it makes our intention clear.

I never use `==`, so I had to run it through some possible edge-case tests to ensure it didn't break:

![image](https://user-images.githubusercontent.com/3948069/132973438-49aa91bb-7f75-4b1f-957d-709ea492c98f.png)

To make it easier on other devs, it's better to always be explicit about this kinda stuff.

Reply:
I always use `== null` for `null | undefined` checks and `=== null` for just `null` checks (which tbh are pretty rare, in 99% of cases you want to check for both). Always seemed pretty clear to me (and to many others, as it's the one exception to the `===` linter checks allowed 😁).

However, I'm trying to keep to your coding style, so will happily make the change.

Reply:
I'm surprised the linter allows it, and I can I see why it exists so long as it's only going to catch `null` and `undefined` and there's no edge case.

I'd have to ask around to see if others know this syntax too. The first time I saw `!=` instead of `!==` used was at my last job. It's one of those "you have to really know JS to understand this syntax" type of issues and could be something where someone does this without realizing the impact:
```diff
- == null
+ === null
```

It's also really difficult to spot in code reviews in general.

It's not my coding style so much as I'm trying to make the intent obvious.

If I found a good argument that said "everyone in JS knows this thing about `== null` and `!= null`", then I'd change my ways. As it stands, I'm unaware how common this is considering I only stumbled upon it because they didn't use `===` anywhere at my last job. Might just be me.